lesson09/main.js

```diff
@@ -31,25 +31,21 @@ function renderMenus(menus){
   ul.appendChild(fragment);
 }
 
-function getMenus() {
+const getMenus = new Promise((resolve)=> {
   renderCircle();
   const menus = [ 
     {to: "bookmark.html", img:"img/1.png", alt:"画像1", text: "ブックマーク"}, 
     {to: "message.html", img:"img/2.png", alt:"画像2", text: "メッセージ"} 
   ]; 
   setTimeout(() => {
-		console.log(menus);
-    return menus;
+    resolve(menus);
   }, 3000);
-}
+})
 
-async function men() {
-	 await getMenus();
-	 removeCircle();
+async function deployMenus() {
+	const menus = await getMenus;

```

Comment:
IMO
Since the name of this variable is the same as the name of the array, why not make it value, etc., as the value received?
```suggestion
	const value = await getMenus;

```

Reply:
Thank you sen-san :)
It's easy to understand for me so I use that.